docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -457,6 +457,29 @@ and a file system location of ``/var/my_tables/test_table``::
 
 .. _iceberg-metadata-tables:
 
+.. _iceberg_metadata_columns:
```

Comment:
`.. _iceberg-metadata-tables:` should be put before `Metadata tables`

Reply:
update @ebyhr